benchmarks/run_mo_kurobako.py

```diff
@@ -0,0 +1,112 @@
+import argparse
+import os
+import subprocess
+
+
+def run(args: argparse.Namespace) -> None:
+    kurobako_cmd = os.path.join(args.path_to_kurobako, "kurobako")
+    subprocess.run(f"{kurobako_cmd} --version", shell=True)
+
+    if not (os.path.exists(args.data_dir) and os.path.isdir(args.data_dir)):
+        raise ValueError(f"Data directory {args.data_dir} cannot be found.")
+
+    os.makedirs(args.out_dir, exist_ok=True)
+    study_json_filename = os.path.join(args.out_dir, "studies.json")
+    subprocess.check_call(f"echo >| {study_json_filename}", shell=True)
+    solvers_filename = os.path.join(args.out_dir, "solvers.json")
+    subprocess.check_call(f"echo >| {solvers_filename}", shell=True)
+    problems_filename = os.path.join(args.out_dir, "problems.json")
+    subprocess.check_call(f"echo >| {problems_filename}", shell=True)
+
+    # Create ZDT problems.
+    cmd = f"{kurobako_cmd} problem-suite zdt | tee -a {problems_filename}"
+    subprocess.run(cmd, shell=True)
+
+    # Create NAS bench problem(A) (for Multi-Objective Settings).
+    dataset = os.path.join(args.data_dir, "nasbench_full.bin")
+    cmd = (
+        f'{kurobako_cmd} problem nasbench "{dataset}" '
+        f"--metrics params accuracy | tee -a {problems_filename}"
+    )
+    subprocess.run(cmd, shell=True)
+
+    # Create solvers.
+    sampler_list = args.sampler_list.split()
+    sampler_kwargs_list = args.sampler_kwargs_list.split()
+
+    if len(sampler_list) != len(sampler_kwargs_list):
+        raise ValueError(
+            "The number of samplers does not match the given keyword arguments. \n"
+            f"sampler_list: {sampler_list}, sampler_kwargs_list: {sampler_kwargs_list}."
+        )
+
+    for sampler, sampler_kwargs in zip(sampler_list, sampler_kwargs_list):
+        name = f"{args.name_prefix}_{sampler}"
+        python_command = f"{args.path_to_create_study} {sampler} {sampler_kwargs}"
+        cmd = (
+            f"{kurobako_cmd} solver --name {name} command python {python_command}"
+            f"| tee -a {solvers_filename}"
+        )
+        subprocess.run(cmd, shell=True)
+
+    # Create study.
+    cmd = (
+        f"{kurobako_cmd} studies --budget 10 "
```

Comment:
We should use the same value as the single-objective one.
```suggestion
        f"{kurobako_cmd} studies --budget 80 "
```